asmcli/lib/installation_dependencies.sh

```diff
@@ -320,6 +307,7 @@ register_cluster() {
   local FLEET_ID; FLEET_ID="$(context_get-option "FLEET_ID")"
   local KCF; KCF="$(context_get-option "KUBECONFIG")"
   local KCC; KCC="$(context_get-option "CONTEXT")"
+  local OPENSHIFT_CLUSTER; OPENSHIFT_CLUSTER="$(context_get-option "OPENSHIFT_CLUSTER_SETUP")"
```

Comment:
I don't see where you set this variable. You might need to call `context_set-option` to set it somewhere and maybe add it to the [context file](https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/main/asmcli/lib/context.sh#L68) to get a default env var value.